gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/task/COPCMessageHandlerTest.java

```diff
@@ -1280,7 +1280,7 @@ public void When_HandleMessage_With_AttachmentProcessingErrorNotStorageCause_Exp
             assertThat(nackMessageDataCaptor.getValue().getNackCode()).isEqualTo(LARGE_MESSAGE_ATTACHMENTS_NOT_RECEIVED.getCode());
 
             verify(migrationStatusLogService, times(1))
-                .addMigrationStatusLog(LARGE_MESSAGE_ATTACHMENTS_NOT_RECEIVED.getMigrationStatus(), CONVERSATION_ID, null);
+                .addMigrationStatusLog(LARGE_MESSAGE_ATTACHMENTS_NOT_RECEIVED.getMigrationStatus(), CONVERSATION_ID, null, "31");
```

Comment:
What's interesting here is that `LARGE_MESSAGE_ATTACHMENTS_NOT_RECEIVED` already has the code associated with it.

Perhaps this suggests we should have a new method called `.addNackMigrationStatusLog` which takes in a `NACKReason`. Thoughts?